src/Microsoft.AspNetCore.Razor.Evolution/Legacy/CSharpCodeParser.cs

```diff
@@ -1590,7 +1632,7 @@ private void HandleDirective(DirectiveDescriptor descriptor)
                     {
                         Context.ErrorSink.OnError(
                             CurrentStart,
-                            LegacyResources.FormatUnexpectedDirectiveLiteral(descriptor.Name, Environment.NewLine),
+                            LegacyResources.FormatUnexpectedDirectiveLiteral(descriptor.Name, LegacyResources.ErrorComponent_Newline),
```

Comment:
The issue here is that `\r\n` doesn't play well with error messages. Now it will say `expected 'new line'`